literature.bib

```diff
@@ -297,6 +311,17 @@ @inproceedings{SBE+:EASE22
 	year = 2022
 }
 
+@mastersthesis{Schiessl22,
+	author = {Schiessl, Dominik},
+	title = {{An Incremental \#SAT Solver for Efficient Analysis of Feature Models}},
+	school = UniUlm,
+	address = {Germany},
+	type = {Master's Thesis},
+	typo3Tags = {SupervisorCS, SupervisorTT, FMCounting},
+	month = JAN,
+	year = 2022
+}
```

Comment:
To appear?

Reply:
Publications that are flagged as "to appear" should be at the top of the file.